arch/arm/src/tms570/tms570_lowputc.c

```diff
@@ -272,19 +273,20 @@ void tms570_lowsetup(void)
 #endif
 }
 
-/************************************************************************************
+/****************************************************************************
  * Name: tms570_sci_configure
  *
  * Description:
  *   Configure an SCI for non-interrupt driven operation
  *
- ************************************************************************************/
+ ****************************************************************************/
 
-int tms570_sci_configure(uint32_t base, FAR const struct sci_config_s *config)
+int tms570_sci_configure(uint32_t base,
+                         FAR const struct sci_config_s *config)
 {
-  float32 divb7;
+  float_t divb7;
   uint32_t intpart;
```

Comment:
isn't it better to use just float in C code?

Reply:
Fixed here: #1366